docs/content/docs/deployment/speculative_execution.md

```diff
@@ -62,6 +62,14 @@ To make speculative execution work better for different jobs, you can tune below
 - [`execution.batch.speculative.max-concurrent-executions`]({{< ref "docs/deployment/config" >}}#execution-batch-speculative-speculative-max-concurrent-e)
 - [`execution.batch.speculative.block-slow-node-duration`]({{< ref "docs/deployment/config" >}}#execution-batch-speculative-speculative-block-slow-node)
 
+Currently, speculative execution uses the slow task detector based on execution time to detect slow tasks. 
+The detector will periodically count all finished executions, if the finished execution ratio reaches the threshold, 
+the median of the tasks' execution time will be defined as the baseline and the execution will 
+be detected as a slow task if its execution time exceeds the baseline. It is worth mentioning that 
+the execution time will be weighted with the input bytes of the execution vertex, so the executions 
+with large data volume differences but close computing power will not be detected as a slow task, 
+when data skew occurs. That will avoid starting invalid attempts.
```

Comment:
That will avoid starting invalid attempts. -> This helps to avoid starting unnecessary speculative attempts.